zipline/protocol.py

```diff
@@ -225,14 +227,17 @@ def __repr__(self):
 
 
 class Position(object):
-
-    def __init__(self, sid):
-        self.sid = sid
+    @expect_types(asset=Asset)
+    def __init__(self, asset):
+        self.asset = asset
         self.amount = 0
         self.cost_basis = 0.0  # per share
         self.last_sale_price = 0.0
         self.last_sale_date = None
 
+        # for backwards compatibility
+        self.sid = asset
```

Comment:
Is there a reason we're aliasing this here but making it a property in https://github.com/quantopian/zipline/pull/1757/files#diff-506712f0fea50ba9a660883d95d114d4R94 ?